### PR TITLE
small ui fixes and reply button replies again

### PR DESCRIPTION
**Pull request by jenbeckett**
Changed message button CSS on focus to not have the tiny bit of blue background 
Added back in the "Reply" button functionality that was mistakenly removed from a previous PR
Added <span> tags on messaging buttons, so whitespace would no longer be underlined 

I left 'Notifications' and 'Messages' tab words bold - as I think they look better but open to suggestions on this

### CODE REVIEW

File: frontend/src/components/messages/MessagesTab.vue [hunk 1]
```diff
@@ -5,15 +5,15 @@
         <v-col cols="12">
           <AppButton size="small" class="messages-button" @click="toggleNewRequestDialog()">
             <v-icon left>mdi-email-plus-outline</v-icon>
-            New message
+            <span>New message</span>
```
Comment (vietle-cgi) [inline frontend/src/components/messages/MessagesTab.vue:8]: Hi Jen, could we remove the span tag in Notification's buttons as well so they will look consistent?

File: frontend/src/components/messages/MessagesTab.vue [hunk 2]
```diff
@@ -91,4 +91,9 @@ export default {
 .messages-button {
   display: inline-block;
 }
+
+.messages-button:focus {
```
Comment (vietle-cgi) [inline frontend/src/components/messages/MessagesTab.vue:95]: Hi Jen, could we add this style for Notification's buttons as well so they will look consistent?

**Review by vietle-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/messages/MessagesTab.vue (hunk 1 above)
Comment (jenbeckett) [reply frontend/src/components/messages/MessagesTab.vue:8]: done :)

File: frontend/src/components/messages/MessagesTab.vue (hunk 2 above)
Comment (jenbeckett) [reply frontend/src/components/messages/MessagesTab.vue:95]: done :)

**Review by vietle-cgi — APPROVED** [review]

**Pull request merged**